clients/src/main/java/org/oracle/okafka/common/network/AQClient.java

```diff
@@ -80,6 +80,7 @@ public abstract class AQClient {
 	private Map<Integer, Timestamp> instancesTostarttime;
 	public List<Node> all_nodes = new ArrayList<>();
 	public List<PartitionInfo> partitionInfoList = new ArrayList<>();
+	private static int userQueueShardsQueryIndex = 0;
```

Comment:
We don't want this static. Same application can connect to 23.4 ai and db > 23.5